pymc3/step_methods/smc.py

```diff
@@ -61,12 +61,12 @@ class SMC:
     A summary of the algorithm is:
 
      1. Initialize \beta at zero and stage at zero.
-     2. Generate N samples S_{\beta} from the tempered posterior (because \beta = 0 this is the
-        prior).
+     2. Generate N samples S_{\beta} from the prior (because when \beta = 0 the tempered posterior
+        is the prior).
      3. Increase \beta in order to make the effective sample size equals some predefined value
         (we use N*t, where t is 0.5 by default).
-     4. Compute a set of N weights W. The weights are computed according to the new
-        tempered posterior.
+     4. Compute a set of N impotance weights W. The weights are computed as the ratio of the
+        probabilities of a sample at stage i+1 and stage i.
```

Comment:
what is the probabilities here - is that the model likelihood you are referring to?

Reply:
Is the likelihood times the prior, but you can simplify that expression because the prior is the same in the numerator and denominator. Thus we can say we are comparing the likelihooods. I can change it by likelihood if you want.

Reply:
changed it

Reply:
Is `impotance` the right word?